model/src/main/resources/META-INF/openapi.yaml

```diff
@@ -174,3 +174,18 @@ components:
               additionalProp2: "yyy"
               additionalProp3: "zzz"
             signedOffBy: "signedOffByName <signedOffBy@example.com>"
+
+    referencesResponse:
+      value:
+        token: "xxx"
+        hasMore: false
```

Comment:
nit: if `hasMore` is `false`, shouldn't the `token` be `null` or not present?

Reply:
Can we have two examples: 1) for a paginated response 2) for a non-paginated (or last page) response?

Reply:
I set the token to `false`, since pagination is currently not supported due to https://github.com/projectnessie/nessie/blob/0e0b8713efca8050b3eefc377c0ee910f770b08d/servers/services/src/main/java/org/projectnessie/services/impl/TreeApiImpl.java#L97